datalake/src/main/java/com/sequenceiq/datalake/service/upgrade/SdxRuntimeUpgradeService.java

```diff
@@ -174,7 +174,7 @@ private FlowIdentifier triggerDatalakeUpgradeFlow(SdxUpgradeRequest request, Sdx
     }
 
     private void validateRollingUpgrade(SdxUpgradeRequest request, SdxCluster cluster) {
-        if (Boolean.TRUE.equals(request.getRollingUpgradeEnabled()) && !SdxClusterShape.MEDIUM_DUTY_HA.equals(cluster.getClusterShape())) {
+        if (Boolean.TRUE.equals(request.getRollingUpgradeEnabled()) && !cluster.getClusterShape().isHA()) {
             String message = String.format("The rolling upgrade is not allowed for %s cluster shape. "
                     + "If you want to use rolling upgrade you need to launch a Data Lake with %s cluster shape.",
                     cluster.getClusterShape().name(), SdxClusterShape.MEDIUM_DUTY_HA.name());
```

Comment:
Mostly a nit... 

This message still doesn't make sense to me. I know an overall fix to define the OS RU for the Scalable shape will go in separately.

Until then: 
1) The code allows an RU for the scalable shape
2) Logs that the shape must be MEDIUM DUTY to perform an RU.